rest_framework_jwt/models.py

```diff
@@ -1 +1,16 @@
-# Just to keep things like ./manage.py test happy
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class Device(models.Model):
+    """
+    Device model used for permanent token authentication
+    """
+    permanent_token = models.CharField(max_length=255, primary_key=True)
+    jwt_secret = models.CharField(max_length=255)
+    created = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    name = models.CharField(_('Device name'), max_length=255)
+    details = models.CharField(_('Device details'), max_length=255, blank=True)
+    last_request_datetime = models.DateTimeField(auto_now=True)
```

Comment:
GitHub does not show the last empty line as far as I remember. The flake8 task in tox did not complain, see: https://travis-ci.org/ArabellaTech/django-rest-framework-jwt/jobs/256993179